HarmonyCoreEF/Query/Internal/HarmonyQueryableMethodTranslatingExpressionVisitor.cs

```diff
@@ -1398,8 +1398,10 @@ protected override ShapedQueryExpression TranslateThenBy(ShapedQueryExpression s
             {
                 return null;
             }
-
-            inMemoryQueryExpression.FindServerExpression().OrderByExpressions.Add(Tuple.Create<Expression, bool>(keySelector.Body, ascending));
+            // This meant to eliminate an additional order by on a key entity generated in the case when ODATA query contains $top and $orderbys
+            if (false)
```

Comment:
this if(false) looks like exploration code rather than something you want checked in. Is there something different you want to do with it?